documentation/leo/03_language.md

```diff
@@ -423,12 +424,24 @@ If the value at `addr` does not exist, then the program will fail to execute.
 A get command that initializes the mapping in case of failure, e.g.
 `let current_value: u64 = Mapping::get_or_init(counter, addr, 0u64);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`.
-If the key is not present, `0u64`  is stored in `counter` and stored in `current_value`.
+If the key is not present, `0u64` is stored in `counter` and stored in `current_value`.
 
 #### set
 A set command, e.g. `Mapping::set(counter, addr, current_value + 1u64);`
 Sets the `addr` entry as `current_value + 1u64` in `counter`.
 
+## For Loops
+
+For Loops are declared as `for {loop variable} in {lower bound}..{upper bound}`.  Currently, the
+loop variable supports unsigned integer types u8, u16, u32, or u64. The lower bound must be
```

Comment:
```suggestion
For Loops are declared as `for {loop variable} in {lower bound}..{upper bound}`.  Typical Leo loop variable types include `u8`, `u16`, and `u32`. The lower bound must be
```

Reply:
What is a `{loop variable}` it is not defined in the documentation.

Reply:
That description still does not help the programmer write a loop @moAlobaidi.
Loops in Leo require a variable name and a variable type.
`{loop variable}` should be defined as `{name: type}` with an example.

Reply:
Okay updated.